ISSUE_TEMPLATE.md

```diff
@@ -3,7 +3,7 @@ Thank you for contributing!
 
 Have a usage question?
 ======================
-Questions *might* be closed without comment, sometimes we just don't have time for them. Before you ask one, here are some resources to get help first:
+The issue tracker isn't the best place for usage questions. This format is not well-suited for Q&A, and questions here don't have as much visibility as they do elsewhere. Before you ask a question, here are some resources to get help first:
```

Comment:
I feel like this is a bit more gentle – "you should go to other places because they're better".